cabal.project

```diff
@@ -3,3 +3,7 @@ packages:
     cryptol-remote-api
     tests
     deps/argo/argo
+
+allow-newer:
+  -- https://github.com/GaloisInc/test-lib/pull/3
+  test-lib:base
```

Comment:
This PR is marked as a draft pending GaloisInc/test-lib#3 being released to Hackage.